docs/01-programs/02-candy-machine/10-how-to-guides/00-my-first-candy-machine.md

```diff
@@ -0,0 +1,600 @@
+# How to create a Candy Machine v3
+
+The goal of this tutorial is to take you from zero to one: you will learn to install Sugar and use its basic commands to configure and deploy a candy machine to Solana's devnet. It will provide you with a foothold on the basics and enough knowledge that you can then use the rest of the developer resources to learn the more advanced details for using Sugar.
+
+## Prerequisite Knowledge
+
+- You should have a basic understanding of how to find and use a terminal on your OS, including navigating directories, and running commands: an example of a terminal for macOS is [iTerm2](https://iterm2.com/).
+- You should have basic familiarity with what Solana is but don't need advanced technical knowledge.
+- You should have basic familiarity with the Metaplex Standard but, again, do not need advanced technical knowledge.
+
+## Setup
+
+This tutorial targets macOS, Linux, and Windows Subsystem Linux (WSL), but all commands and screenshots are from a Linux system. It should work on any of those three systems. For installing on Windows see the Windows method on the [installation page](/developer-tools/sugar/overview/installation).
+
+### Install the Solana CLI Tool Suite
+
+The [Solana CLI Tool Suite](https://docs.solana.com/cli/install-solana-cli-tools) is useful to have for Solana development, and we will use the config file to store our keypair file path and RPC node url to simplify the Sugar commands we run, by eliding those options.
+
+:::info
+To install, follow [this guide](../../../guides/cli-wallet#setting-up-a-devnet-wallet-for-testing) to install the Solana Tool Suite and set up a devnet wallet.
+:::
```

Comment:
This doesn't need to be an "info" callout.

If you want to attract the user's attention to the installation link, I would make this part of the sentence in bold: "**follow this guide to install the Solana Tool Suite**".

It's easy to think that callouts "call" for the user's attention but in reality, they are more seen as "optional reading" and will often be skimmed through.

When too many are provided, they distract the reader during their journey.